cpp/open3d/geometry/VoxelGrid.h

```diff
@@ -259,6 +260,9 @@ class VoxelGrid : public Geometry3D {
     double voxel_size_ = 0.0;
     /// Coordinate of the origin point.
     Eigen::Vector3d origin_ = Eigen::Vector3d::Zero();
+    /// Orientation (rotation) of the voxel grid (world = origin_ +
+    /// origin_rotation_ * local)
+    Eigen::Matrix3d origin_rotation_ = Eigen::Matrix3d::Identity();
```

Comment:
This is the rotation of the voxel_grid frame.  Good names would be `voxel_grid_origin_` and `voxel_grid_rotation`. Since we named it `origin_`, lets name the rotation simply as `rotation_`. Rotation of origin is confusing, since we cannot rotate a point.